SU-SWS/stanford_profile_helper
# D8CORE-2765: removing action link and changing colors for mailto links

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
Color fixes for the mailto and external links.

# Needed By (Date)
Not urgent

# Urgency


# Steps to Test

1. Pull this branch
1. clear cache
1. Verify that the external and mailto links behave as expected.


![Screen Shot 2020-11-13 at 9 41 04 PM](https://user-images.githubusercontent.com/284440/99140812-dc6ac000-25f9-11eb-9466-35fabbab9485.png)

# Affected Projects or Products
D8Core

# Associated Issues and/or People
- JIRA ticket: 
-- https://stanfordits.atlassian.net/browse/D8CORE-2765
-- https://stanfordits.atlassian.net/browse/D8CORE-2764
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
